Nos/Models/ReportCategory.swift

```diff
@@ -79,6 +85,13 @@ enum ReportCategoryType {
             ReportSubCategoryType.malware,
         ]
     )
+
+    static let nsfw = ReportCategory(
+        name: .moderation.nsfw,
+        code: "NW",
```

Comment:
I debated what to use for this code; unfortunately, "NS" is already used.

Reply:
@dcadenas unfortunately I don't see this category in the [draft](https://github.com/rabble/nips/blob/724e05e762a634e501bdcf6cbefaa91f99b1903b/69.md) or [cleanstr reports](https://github.com/planetary-social/cleanstr/blob/main/src/lib/openAICategories.js).

Reply:
They are not fully expressive for what we try to map. NS is here https://github.com/planetary-social/cleanstr/blob/main/src/lib/openAICategories.js#L44-L49 but of course not the same. And on top of that, the report categories don't perfectly match either. 

IMO we should have a vocabulary that is derived from the top nip-56 categories for our use case. I'd create the vocabulary based on how we use it (just like normal language) instead of a language that we force ourself to map to (that's why esperanto never worked)